using-datapower-in-kubernetes/getting-started-datapower-in-gke.md

```diff
@@ -0,0 +1,350 @@
+number of pod "replicas" are running at any given time. Below is the sample *deployment* YAML for the *datapower* application
+ we use for this tutorial:
+
+```
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: datapower
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: datapower
+        track: stable
+    spec:
+      containers:
+      - name: datapower
+        image: "ibmcom/datapower:latest"
+        stdin: true
+        tty: true
+        ports:
+        - name: web-mgmt
+          containerPort: 9090
+        env:
+        - name: DATAPOWER_ACCEPT_LICENSE
+          value: "true"
+        - name: DATAPOWER_INTERACTIVE
+          value: "true"
+        volumeMounts:
+        - mountPath: /drouter/config/foo/foo.cfg
+          name: config-volume
+          subPath: foo.cfg
+        - mountPath: /drouter/config/auto-startup.cfg
+          name: config-volume
+          subPath: auto-startup.cfg
+        - mountPath: /drouter/config/auto-user.cfg
+          name: config-volume
+          subPath: auto-user.cfg
+        - mountPath: /drouter/local/foo
+          name: local-foo-volume
+      volumes:
+      - name: config-volume
+        configMap:
+          name: datapower-config
+      - name: local-foo-volume
+        configMap:
+          name: datapower-local-foo
+```
+
+Note that we use the `datapower-config` and `datapower-local-foo` ConfigMaps as a `volumes` parameter and mount it in the `volumeMounts` section.
+The `mountPath: /drouter/config` is special in this case because DataPower will automatically execute a config file named
+*auto-startup.cfg* if found in the */drouter/config* directory, which we have placed in this configMap. 
+
+There is likewise a *deployment* file for our *backend* application which can be found under `kubernetes/deployments/backend-deployment.yaml`
+
+To create the deployments we head over to the `kubernetes/deployments` directory and issue the following commands:
+
+`$ kubectl create -f datapower-deplyment.yaml`
+
+`$ kubectl create -f backend-deployment.yaml`
+
+### Exposing Our Application to the Internet
+
+Before we continue, let's stop and reflect for a minute on what we have accomplished so far. At this time, we have deployed an applicaiton
+ that can recover from a failure in one of our nodes in our Kubernetes cluster. We can deploy this applicaiton in a highly reproducible way.
+ Additionally, the application's configuration is managed independently of the application and can be changed easily without having to rebuild our applicaiton.
+
+Now, we want to expose the application service so that we can access it from the public internet. 
+We can expose the datapower multi-protocol gateway service running on port 80 that was configured using our  `datapower-config` ConfigMap by 
+heading over to the `kubernetes/services` directory and running:
+
+`$ kubectl create -f datapower-service.yaml`
```

Comment:
You need a second command to create the backend:
kubectl create -f backend-service.yaml